legal-api/src/legal_api/resources/v2/business/business_furnishings/furnishing_documents.py

```diff
@@ -0,0 +1,65 @@
+# Copyright © 2024 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Retrieve the specified lettter for the furnishing entry."""
+from http import HTTPStatus
+from typing import Final
+
+from flask import jsonify, request
+from flask_cors import cross_origin
+
+from legal_api.exceptions import ErrorCode, get_error_message
+from legal_api.models import Business, Furnishing
+from legal_api.reports.report_v2 import ReportTypes, ReportV2
+from legal_api.services import authorized
+from legal_api.utils.auth import jwt
+
+from ..bp import bp
+
+
+FURNISHING_DOC_BASE_ROUTE: Final = '/<string:identifier>/furnishings/<string:furnishing_id>/document'
+
+
+@bp.route(FURNISHING_DOC_BASE_ROUTE, methods=['GET', 'OPTIONS'])
+@cross_origin(origins='*')
+@jwt.requires_auth
+def get_furnishing_document(identifier: str, furnishing_id: int):
```

Comment:
I think for now maybe we lock this down to only staff or system roles.

Emailer, the furnishings job and in the future, maybe admin screens will use this endpoint.

If we ever need to let the user access this endpoint, we can open it up then.